CIP5/README.md

```diff
@@ -23,31 +23,58 @@ Therefore, we can leverage bech32 for binary data encoding, with a set of common
+| `root_sk`    | CIP-1852's root private key             | Ed25519 private key                |
+| `root_vk`    | CIP-1852's root public key              | Ed25519 public key                 |
+| `root_xsk`   | CIP-1852's extended root private key    | Ed25519-bip32 extended private key |
+| `root_xvk`   | CIP-1852's extended root public key     | Ed25519 public key with chain code |
+| `script_sk`  | Script signing key                      | Ed25519 private key                |
+| `script_vk`  | Script verification key                 | Ed25519 public key                 |
+| `script_xsk` | Extended script signing key             | Ed25519-bip32 extended private key |
+| `script_xvk` | Extended script verification key        | Ed25519 public key with chain code |
+| `stake_sk`   | Stake address signing key               | Ed25519 private key                |
+| `stake_vk`   | Stake address verification key          | Ed25519 public key                 |
+| `stake_xsk`  | Extended stake address signing key      | Ed25519-bip32 extended private key |
+| `stake_xvk`  | Extended stake address verification key | Ed25519 public key with chain code |
+| `vrf_sk`     | VRF signing key                         | VRF signing key                    |
+| `vrf_vk`     | VRF verification key                    | VRF verification key               |
+
+#### Hashes
+
+| Prefix       | Meaning                                       | Contents                                             |
+| ---          | ---                                           | ---                                                  |
+| `addr_vkh`   | Address verification key hash                 | blake2b\_224 digest of a payment verification key    |
+| `pool`       | Pool operator verification key hash (pool ID) | blake2b\_224 digest of an operator verification key  |
+| `script_vkh` | Script verification key hash                  | blake2b\_224 digest of a script verification key     |
+| `stake_vkh`  | Stake address verification key hash           | blake2b\_224 digest of a delegation verification key |
+| `vrf_vkh`    | VRF verification key hash                     | blake2b\_256 digest of a VRF verification key        |
+
+
+#### Miscellaneous 
+
+| Prefix       | Meaning               | Contents                                                      |
+| ---          | ---                   | ---                                                           |
+| `addr`       | Mainnet address       | Network tag, payment credential and optional stake credential |
+| `addr_test`  | Testnet address       | Network tag, payment credential and optional stake credential |
+| `stake`      | Mainnet stake address | Network tag and stake credential                              |
+| `stake_test` | Testnet stake address | Network tag and stake credential                              |
+| `script`     | Script hash           | blake2b\_224 digest of a serialized transaction script        |
```

Comment:
Shouldn't we also move this under the "Hashes" heading? Then we could change `#### Miscellaneous` to `#### Addresses`.